### ISSUE TITLE
Add a command to queue media

### ISSUE THREAD

**Issue by ayonix** [metal3d/idok#11]
Being able to queue items (for me especially youtube videos) would be a nice addition.

**Comment by metal3d** [comment]
That's absolutely right. I already check how I can do this feature and I've got many problems with it.
I will try to continue this feature in a couple of days.

**Comment by 97-109-107** [comment]
Do you have any WIP code on this? Any particular areas that gave you trouble?

**Comment by metal3d** [comment]
Right now... No.
I need time and unfortunatly I'm very busy with m'y job. But I've got
ideas. I know you want me to have that feature and you can trust me, I'm
sorry to be unable tout work on idok at this time.
As soon as I can, I will provide a testing realease.
Really sorry.

Le dim. 12 mars 2017 10:43, Antoni Kaniowski <notifications@github.com> a
écrit :

**Comment by 97-109-107** [comment]
_You really ought not to be sorry, it is us, the users, who are grateful for having this tool!_
